docs/add-scalardb-to-your-build.md

```diff
@@ -6,7 +6,7 @@ You can install it in your application using your build tool such as Gradle and
 To add a dependency on ScalarDB using Gradle, use the following:
 ```gradle
 dependencies {
-    implementation 'com.scalar-labs:scalardb:3.8.0'
+    implementation 'com.scalar-labs:scalardb:3.9.1-SNAPSHOT'
```

Comment:
This snapshot version doesn't seem to be published. `3.9.0` is better?

```
> Task :compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileJava'.
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not find com.scalar-labs:scalardb:3.9.1-SNAPSHOT.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/com/scalar-labs/scalardb/3.9.1-SNAPSHOT/maven-metadata.xml
       - https://repo.maven.apache.org/maven2/com/scalar-labs/scalardb/3.9.1-SNAPSHOT/scalardb-3.9.1-SNAPSHOT.pom
     Required by:
         project :
```

Reply:
Good point. To use SNAPSHOT versions, we need to add the following snapshot maven repository to `build.gradle`:

```gradle
    maven {
        url 'https://oss.sonatype.org/content/repositories/snapshots/'
    }
```

Reply:
Should we mention `3.9.0` instead? If so, I think we should update the [README.md](https://github.com/scalar-labs/scalardb/blob/3.9/README.md) as well since it mentions using `3.9.1-SNAPSHOT`.

Reply:
@josh-wong This page is used in the Doc site, right? If so, we should show the released version `3.9.0` instead of the SNAPSHOT version `3.9.1-SNAPSHOT`.

And, yes, we should update README as well.

Reply:
Thanks for confirming! I've updated both docs (including the README) to mention `3.9.0`. PTAL!

Reply:
@josh-wong Thank you!

Can you please also update the following files:

- docs/index.md
- docs/getting-started/build.gradle

Thanks.

Reply:
@brfrn169 Done!

Should we update `https://github.com/scalar-labs/scalardb/blob/docs/3.9/fix-version/build.gradle` as well to `3.9.0`? Or should that stay as is?

Reply:
@josh-wong Thank you!

We don't need to update `https://github.com/scalar-labs/scalardb/blob/docs/3.9/fix-version/build.gradle`. Thanks.

Reply:
Got it. Thank you for confirming!